MPU/src/cascadiamc.cpp

```diff
@@ -26,9 +19,7 @@ void CASCADIAMC::writeMCState()
     if(isMCLocked)
     {
         disableMCLockout();
-        while(!motorCommand_wait.isTimerExpired()){}
     }
-    while(!motorCommand_wait.isTimerExpired()){}
```

Comment:
Why are you removing all these delays completely? I thought the goal was to make them non-blocking, not non-existant